src/coreclr/pal/src/cruntime/misc.cpp

```diff
@@ -108,155 +108,27 @@ __iscsym( int c )
     PERF_EXIT(__iscsym);
     return 0;
 }
-
-
-/*++
-
-Function :
-
-    PAL_errno
-
-    Returns the address of the errno.
-
---*/
-int * __cdecl PAL_errno( int caller )
-{
-    int *retval;
-    PERF_ENTRY(errno);
-    ENTRY( "PAL_errno( void )\n" );
-    retval = (INT*)(&errno);
-    LOGEXIT("PAL_errno returns %p\n",retval);
-    PERF_EXIT(errno);
-    return retval;
-}
-
-
-/*++
-Function:
-
-   rand
-
-   The RAND_MAX value can vary by platform.
-
-See MSDN for more details.
---*/
-int
-__cdecl
-PAL_rand(void)
-{
-    int ret;
-    PERF_ENTRY(rand);
-    ENTRY("rand(void)\n");
-
-    ret = (rand() % (PAL_RAND_MAX + 1));
-
-    LOGEXIT("rand() returning %d\n", ret);
-    PERF_EXIT(rand);
-    return ret;
-}
-
-
 /*++
-Function:
 
-   time
+PAL forwarders for standard macro headers.
 
-See MSDN for more details.
 --*/
-PAL_time_t
-__cdecl
-PAL_time(PAL_time_t *tloc)
+PALIMPORT DLLEXPORT int * __cdecl PAL_errno()
```

Comment:
Is it non-trivial to delete this for some reason?

Reply:
I missed these when going through the removal. I'll circle back and make sure to remove them.